components/avatar/avatar.vue

```diff
@@ -7,29 +7,36 @@
       AVATAR_SIZE_MODIFIERS[size],
       AVATAR_COLOR_MODIFIERS[color],
       avatarClass,
+      'd-ps-relative',
```

Comment:
Had to add `relative` class here because I'm absolutely positioning presence - do we need to update the `dt-avatar` DT class to include the `position: relative` attribute? That would be the way to go

Reply:
Yep I agree with this.